x/lockup/client/cli/cli_test.go

```diff
@@ -735,38 +735,6 @@ func (s IntegrationTestSuite) TestCmdAccountLockedLongerDurationDenom() {
 	}
 }
 
-// TestGetCmdParams tests module params CLI query commands
```

Comment:
Is the plan to also generate the respective cli tests in the future as well?

Reply:
We need an entirely new form of CLI tests, the current ones are pretty useless integration tests.